package.json

```
@@ -111,7 +111,7 @@
     "@fastify/type-provider-typebox": "^3.3.0",
     "@hyperswarm/secret-stream": "^6.1.2",
     "@mapeo/crypto": "1.0.0-alpha.10",
-    "@mapeo/schema": "3.0.0-next.13",
+    "@mapeo/schema": "^3.0.0-next.14",
```

This should stay pinned

```suggestion
    "@mapeo/schema": "3.0.0-next.14",
```

---

Done in 3b6069776123445b959de85c610c473785af016f.